docs/dev-setup-non-vagrant.md

```diff
@@ -342,7 +342,7 @@ fi
 ./tools/do-destroy-rebuild-test-database
 ./manage.py compilemessages
 sudo ./scripts/lib/install-node
```

Comment:
It's nice for it to be possible to call it directly for debugging, but I think as long as we provide a path they can run it from, that's probably fine.  And yeah, we can do what we've been doing with `node` and `npm` of installing a wrapper in `/usr/bin`.